### PR TITLE
feat: #67 rp adapter creates public DIDs for tenants

### LINKED ISSUE #67: RP Adapter: createRPTenant should create tenant publicDID
The `createRPTenantEndpoint` should create publicDIDs for the tenants. This aligns with the principle of "adapters hide all the didcomm/vc complexity".

### PR DESCRIPTION
**Pull request by llorllale**
Depends on #72 
Closes #67

Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

File: pkg/dids/trustbloc_did_creator.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package dids
```
Comment (rolsonquadras) [inline pkg/dids/trustbloc_did_creator.go:6]: This should be `did` instead of `dids`(plural). Same is followed in aries framework as well.

Comment (llorllale) [reply pkg/dids/trustbloc_did_creator.go:6]: It would clash with that package, forcing us to provide aliases.

Comment (rolsonquadras) [reply pkg/dids/trustbloc_did_creator.go:6]: It shouldn't be an issue as we do use aliases in many places.

Comment (fqutishat) [reply pkg/dids/trustbloc_did_creator.go:6]: can we change it

Comment (llorllale) [reply pkg/dids/trustbloc_did_creator.go]: done

**Pull request merged**